frontend/src/views/players/players.ctrl.ts

```diff
@@ -0,0 +1,10 @@
+class PlayersCtrl{
+
+  public getAll = async (useProxy: boolean = true): Promise<any> => {
+    const url = useProxy ? `/api/players` : `http://localhost:5000/players`;
```

Comment:
i would consider using a http **client** passed to the PlayersCtrl or getAll. this will make the useProxy and **/api'** url prefix enclosed.
now you method could look like this if you pass opts to PlayerCtrl

public getAll = ():Promise<any> => this.opts.client.get('/players')

Reply:
Thanks alex! I went quick and dirty to make it happen fetch + jest 👫 I was having a [similar issue to this one](https://github.com/node-fetch/node-fetch/issues/481). I will improve it later on 😉